### ISSUE TITLE
Offline Folder for inventory in neos

### ISSUE THREAD

**Issue by Jazzneo** [Frooxius/NeosPublic#325]
I was wonder if  be possible in future be able save to hard drive in folder in inventory so we still access stuff in offline mode that put in that folder without need go to local world to save there

**Comment by Frooxius** [comment]
Yes. One thing I could actually do is allow accessing your cloud inventory when offline, since there's already support for that as all the items that you load are cached. Would that work well enough?

**Comment by Jazzneo** [comment]
> Yes. One thing I could actually do is allow accessing your cloud inventory when offline, since there's already support for that as all the items that you load are cached. Would that work well enough?

Yea that would work to then people get access to there stuff when then not online

**Comment by alexderpyfox** [comment]
Feature so you can make things 100% local till you want to upload it so if you are building on a project that you save many versions of and then when finished settable to go on the cloud

**Comment by Frooxius** [comment]
@alexderpyfox One thing I should note though is that saving multiple work versions is really cheap on storage. Neos deduplicates assets, so if you have multiple versions of the same thing it usually takes a small amount of extra storage, so that shouldn't be a worry.

**Comment by alexderpyfox** [comment]
Personally I need the feature to just back up all my cloud items to an offline storage so I can keep them there and access them but not being in my cloud storage to make room for new things to build (it's in the case of not being able to pay for storage anymore but still wanting to build things so offloading them and then having a almost empty cloud account again to be able to build things for online)

**Comment by alexderpyfox** [comment]
I see it as like so making it  can't spawn objects from local inventory in other people's worlds but when clicking on a local object in a local inventory that it will give a button that says "upload to neos" button kinda giving it more ability to treat it as unity VR for creators where they can put as much things in Neos as they want without storage issues and only upload things to the cloud on button press
